src/common/Common.baseLogos.ts

```diff
@@ -9,3 +9,6 @@ export const walletTrackerLogo =
UOSxb9WdNOGowFhB06jHhvSRadLf2e0EBbARYWzQd7VUy0GCaLZgSrg36WEHe2LnEDu+QLDX680BhPQMlXOqG+76WLF/hsmUYsSBfPEkGw27/OxchkzOpT4LkST1mHsdVFuIxRKMFQJh+M4ENVhmul0XRaOPSVByOY+qoCFS8TkBki3qwJtb8R2IJHu+nJmAwPNTd5nnmg45MxnKQ7Eb5jwI2Y4BG21bV0RvGyz1t2WeeedWeAmvCKGVTThPMGJYjfd+B3+DC8nVNK/WrpH43iUbmp7q1rnO7RKIFeTnWdjAcK5F3jFcVWfolrapf1OeUmpzroP4D/fzZOYIh0qnbXs3EGGwAC1BWLPBsXXwZCFRcTSlbLamBdYBMWHmLZZ8yV+MOROwMghzufDjrPAr1rVhB9ODKdlm7rfwMu8MZzO0sVTSW+Kdntr4QkE0k5AyylS205Ied3qZ+ONealVz4da8y2uRoEve7pWLc77+iJZKLu+TrtObc7H4/mOcZ0thj2rm++xa5q5imYzsjTezxaFJ3Sj0cnOqoK+SsJhUQWwhYKLc/HPwAWUm5JiT8fH7HklRFL+GklEn/9o8gh/QRMZ148xI3fDbwPLCW8dCGk0ucuTfvGnKGlqyZQcTYlZKs0EwBWNNXgr/UvmCHR3rb1s2evjbCMI/Gcg5pYpJE1jDDDDgEdYI9n9KBuYaeMtGzMybsCZdnFplilVB1X4p2hn6mqlbIhAFi96mDFHIAI5iFr550Nz2vN6/HHw/I+cyk1wd4b/NTc2wMdafMtLPEc/yd6hSJ/eUfzp/3m4OIs8fxCHenTuT5P+WKjPrkAsKZheX7UruwEQb4RwVT+/swRuarpdTxdb/LptRTjQ1UKW10svjyYeYYmnXwso7c20Akt3y6cy89Yby01rsu7Kv4aGAv4lC4UdY6YNmJy7vQ6IxrcvVQ3LxsNxcYCSOQeu2v1Rv97/VL4m6qN+DL9u7I/AmVvNL48MKMSooLLhIO0vOtHSHxtl0niLCpEWbYPP86bZW7wAVWjGD/FW8a1bHMDgAnT0V+Ac6rs5HTAZgL2zuDi+udE9SNfn+0RWsefRGjcphAHRaz1j3YSf9mbnd2pq1Ul7B9FSD1d+eNTkSVffuIURQhZWOwuvY1XWUISrqNLs0Mh5ycAjubtQK8eA8FrzW9kZk9ghrEeaMsG7qZ+c3BR1vmmvvxWAHwTaSKjSgtLG3lFEQIAI7oqtHw4oQrLfNDlQprnLJ/MmL+pBi82+BGkIV5+u6key+zCntnNbGF3+pfnlpre7TMuKRAkUXKGL7d0pwjXUjLWhPyTKcVrIh2lqsvsBOzpORZjcCSVxI4jlYKZkWdh0QIcbR9iidiE4KJid9kaUWJSAGCdyGYW0WOQDf78A4/g/XDfiP9PqeSENg32m2e4RmJewR0yAR2pul7k85RdLzr4rL40AOIgCFW+C0rPlOlYr836wOb4viBCxbOJ6vIgsAmzSoYWOvodO4kTP+l3DSZ5LXz0ipD3fO7Ss2T5NwWApXSpMyfk3AbAstwth9V6PHLoL/gozPaaR045zXUKbu9/A/rZ8qwUalejOzJqFpkl/cWYAgCThJ0MgiHnGgLwpe7g3CluCG7B04eXoFFrslJh3UqLiZqrZIM99X6+V8KTJ42uKLDV445MnkhmmVKQaQAwjVYFlh2lENsmCvKDtgO3Rap+vqECzLdgv8r8U/pOKbTl40LvebgiN+1djJRABPinRtXxJd4Z7LreVLEEAIyDmuaawVBCb1HgB6Y40mnMbhI/DX8OFoLW208LzIQ7v+AeHOc4GlmKeNZvI92xwYfmHlucVSzxYndX6pYBIDETgNhe5l4OjKTt9DsbfHaf8EH4Y/wjZMETcqnuFSR2xyxW73TXSPiyJnEHoLSJx7kssmkfVJ1pxZef6NtSACT2BKGQ810rN4Z6OGFRyU8eWowvol9jr1qr/4K5IMDi1TlPDuzl7iJbIU5wHoef9vs3sP9nuOxyuyNnmV3zO/NoOQBYB/HTQdi1xcojYirlsllhb2w/GrQmPHzwz1KPWYsMHMvJc+eAnyBXyUGRvRA28eQM+t2lmwUIea/RFR5nZrefSsaMACDRWfubhDwq5vz0OEixd40/CG+Pp66pUw9iZ+QL7Int5Q5eZes4S7sywjkM+bYB8a/7dNcpMu/zcnBrVIUs9nlKZV6FNCLcNuFxVZStxMzGGsUKvbsDWZpWtQtqLW5qHkU8zs4qHlLRYbZ9hWCajHlXhLeMzgBtM0FD5Qg4KHMxP0aEuf8TdfWn/m81ok2e4p5uQSKC9FrsFgAwFiprK3NcOZhPKb3ZjFPJEQ4KlRDypMsVvtfqzV6P7AH0Ol0V8I9RgFV87mVWDbd1+4s2jjiPbtwSUNSBqj5f1ox3uNtYULHbZoBkXuM+hg5SBpBFFsjQ50looNe6onQljw+f1cL2CAASQlQFVowlCr0HlJbKCybwdct8tTJteIUhWEE19bdGrtu85GTq9SgA2oBAq1wkFPlaOgzN8kESABV/gvXk8dlL3c5jS0iJ+Bt6MqOcpk2vAADjbx1dZw80N0y22eizVDIqWU9OPWykx4vY4AuORx0l9KomV/T1WWSWKtg2I9V7DQCSpVsdXlmmqepsEJQfAScGlVLqV2y254td05ZlZBRNEO2VAEiWpyroX8LeqVRAPBTsKdPeX1g2Lg00SIAKn6fs6t7McZ9QKMte5higTaAaLQHBz3qFQlOsI5SSxxQFq6MHlQ1Th0w1nwI1w8L2CQAk64BSSmqCK8pbl4fjADq2B5cJFZS+DaJ8QTVtWYm3vBItb4X2mdLnAKCn2TWhV34YpTEWCHkujQdzxa/orJaNghA24IyFjXDTa32knJ1c+nSxWkk9rozVdHWWGmwaRaEMJSCEEC0LUM79V2zq2SBg2SGNYhujIPgUlLwDqG9SqjQTG9EUQndHHLZtU0nvn9ZFBuF/ATQIOV6PBedeAAAAAElFTkSuQmCC';
```

Comment:
@corlard3y this approach was used because for certain channels svg logos were not possible. The logo for the cyber logo is present in figma, please take it from there and remove this.